src/main/java/com/google/api/codegen/viewmodel/DynamicLangXApiView.java

```diff
@@ -68,6 +72,8 @@
 
   public abstract boolean hasPageStreamingMethods();
 
```

Comment:
Why not make this Nullable as well and removing it from the PHP surface transformer.